### PR TITLE
[AAE-3637] Attach file - Upload button is not disabled when the user is in search mode

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/AAE-3637

**What is the new behaviour?**

image

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts [hunk 1]
```diff
@@ -196,6 +196,9 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     @Output()
     siteChange: EventEmitter<string> = new EventEmitter<string>();
 
+    @Output()
+    disableUploadButton: EventEmitter<boolean> = new EventEmitter<boolean>();
```
Comment (arditdomi) [inline lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts:200]: can you use a different name for the event emitter? The panel doesn't know anything about the upload button that is part of the content-node-selector

**Review by arditdomi — CHANGES_REQUESTED** [review]

**Review by popovicsandras — COMMENTED** [review]
Please see Ardit's comment.

File: lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts (hunk 1 above)
Comment (sivakumar414ram) [reply lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts]: Updated

**Review by eromano — COMMENTED** [review]
I am not sure about this behavior

**Review by mauriziovitale — CHANGES_REQUESTED** [review]
Add unit test, please

**Review by arditdomi — CHANGES_REQUESTED** [review]
some documentation is probably missing. You need to add the new Output in a doc file

**Comment by mauriziovitale** [comment]
As discussed with @eromano and @rallegre let's have an additional message that explains the reason why is disabled. We will use the same div to show the message in case the user does not have permission on that folder

**Review by mauriziovitale — APPROVED** [review]

**Review by mauriziovitale — CHANGES_REQUESTED** [review]
change the string

**Pull request merged**